.github/workflows/icu_ci.yml

```diff
@@ -490,11 +490,15 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: bazelbuild/setup-bazelisk@v1
+    - name: Get CI Linux runner VM version
+      id: linux-version
+      run: |
+        echo "LINUX_VERSION=$(cat /etc/os-release | grep VERSION_ID | tr '\"' '\t' | cut -f 2)" >> $GITHUB_OUTPUT
```

Comment:
Simplified:

```
grep -F VERSION_ID /etc/os-release | cut -d'"' -f2
```

Reply:
Done. Thanks.

Reply:
If you want to eliminate the dependency to `cut`, you can do it with only POSIX shell built-ins too:

```sh
. /etc/os-release && echo "LINUX_VERSION=$VERSION_ID"
```

or, if you don't want to source the variables in `/etc/os-release`:

```sh
echo LINUX_VERSION=$(ver=$(grep -F VERSION_ID /etc/os-release); ver=${ver#*\"}; ver=${ver%\"*}; echo $ver)
```